packages/neuron-wallet/tests/services/asset-account-service.test.ts

```diff
@@ -482,4 +482,43 @@ describe('AssetAccountService', () => {
       expect(result.length).toEqual(2)
     })
   })
+
+  describe('Test Token Info List', () => {
+    beforeEach(async () => {
+      const assetAccounts = [
```

Comment:
Since we only test against the `SudtTokenInfo` entity, do we have to persist these `Asset Accounts`? I think we can just persist the `SudtTokenInfo` entities for this test case?

Reply:
Updated in https://github.com/nervosnetwork/neuron/pull/1720/commits/c11d27d991f5183bdb99d6742b102997a46eeb3c